turbopack/crates/turbopack-core/src/chunk/chunk_group.rs

```diff
@@ -149,36 +163,47 @@ pub async fn make_chunk_group(
             .collect(),
     );
 
-    let mut referenced_output_assets = (*external_output_assets.await?).clone();
-    referenced_output_assets.extend(
-        references_to_output_assets(external_module_references)
-            .await?
-            .await?
-            .iter()
-            .copied(),
-    );
-
-    let rebased_modules = traced_modules
+    let traced_output_assets = traced_modules
         .into_iter()
-        .map(|module| {
-            RebasedAsset::new(
-                *module,
-                module.ident().path().root(),
-                module.ident().path().root(),
-            )
-            .to_resolved()
+        .map(|module| async move {
+            Ok(ResolvedVc::upcast(
+                RebasedAsset::new(
+                    *module,
+                    module.ident().path().root(),
+                    module.ident().path().root(),
```

Comment:
When these two arguments are identical, rebasing is a no-op. How does that make sense? In this case we could just omit the `RebasedAsset` and use the `module` directly? cc @mischnic

Reply:
Yes, this is really just converting the `module` into an `OutputAsset`, without any rebasing. We should just use a `RawOutput` asset instead.